lib/templates/accountTemplate.js

```diff
@@ -86,23 +95,59 @@ class AccountTemplate {
     fsUtils.writeConfig(this.TEMPLATE_TYPE, name, templateConfig);
   }
 
-  static #prepareConfigDetails(template, existingConfig = {}) {
+  static #prepareConfigDetails(type, envId, template, existingConfig = {}) {
     const attributes = this.CONFIG_ITEMS.reduce((acc, attribute) => {
       if (template.hasOwnProperty(attribute)) {
         acc[attribute] = template[attribute];
       } else if (existingConfig?.hasOwnProperty(attribute)) {
         acc[attribute] = existingConfig[attribute];
       }
+
       return acc;
     }, {});
+
+    // We need to also create the mapping_list_ranges, which could be different per firm & partner
+    const existingMappingRanges = existingConfig.mapping_list_ranges || [];
+    const updatedMappingRanges = [...existingMappingRanges];
+
+    for (let mappingListRange of template.mapping_list_ranges) {
```

Comment:
Again, just to confirm: we get the latest mapping list ranges from the server, add the new ones to the local version and update the ones that were updated. But should we also remove the ones that no longer exist on the server? :thinking:

Reply:
idd that would also be best to cover; added the logic for this case

Reply:
idd that would also be best to cover; added the logic for this case

Reply:
Again, `mappingListRange` is not saying anything. I'd use `mappingListRangeFromServer`, `serverMappingListRange` or `latestMappingListRange`.